badges.md

```diff
@@ -32,23 +34,41 @@ Translate at least **15,000** words in any of the Qiskit related educational mat
 Earning Criteria <br>
 Translate at least **25,000** words in any of the Qiskit related educational material (example: Qiskit Documentation, Qiskit Tutorials, Qiskit Textbook, etc.).
 
-Note: Your name must be listed under the following [LOCALIZATION_CONTRIBUTORS](https://github.com/qiskit-community/qiskit-translations/blob/master/LOCALIZATION_CONTRIBUTORS) file as proof that you have signed the CLA (Contributor License Agreement) necessary to join the Qiskit Localization Project.
+> **Note:** Your name must be listed in the [LOCALIZATION_CONTRIBUTORS](https://github.com/qiskit-community/qiskit-translations/blob/master/LOCALIZATION_CONTRIBUTORS) list as a proof that you have acknowledged [Qiskit Code of Conduct](https://github.com/Qiskit/qiskit/blob/master/CODE_OF_CONDUCT.md), [Translate Responsibily](https://github.com/qiskit-community/qiskit-translations/blob/master/CODE_OF_CONDUCT.md#translate-responsibly), and accepted the Contributors License Agreement (CLA) upon joining the project.
+
+## Frequently asked Questions
+
+### :question: How can I apply for a badge?
+The organization team will open an application form for all eligible members to apply for their digital badges.
 
-## How can I apply for a badge?
-Badges are issued on a yearly basis. The organization team will open an application form for all eligible members to apply for their digital badges. 
+### :question: Am I eligible if I'm contributing to translate the Qiskit Textbook?
```

Comment:
Maybe
"Am I eligible if I'm contributing to **translating** the Qiskit Textbook?"
instead of 
"Am I eligible if I'm contributing to **translate** the Qiskit Textbook?"

Reply:
Updated in https://github.com/qiskit-community/qiskit-translations/pull/1141/commits/03f96fe043fe7ba90a42ffa44285353799ba6452